language-server/src/server.js

```diff
@@ -257,7 +263,11 @@ const getTokenBuilder = (uri) => {
 
 const buildTokens = (builder, document) => {
   const instance = JsoncInstance.fromTextDocument(document);
-  const dialectUri = instance.get("#/$schema").value();
+  if (instance === undefined) {
+    return;
+  }
+
+  const dialectUri = instance.get("#/$schema")?.value();
```

Comment:
The `?` isn't necessary because `get` always returns an instance even if its value is `undefined`.